crates/router/src/connector/globalpay/transformers.rs

```diff
@@ -23,30 +24,196 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for GlobalpayPaymentsRequest {
+                        enums::CaptureMethod::Manual => requests::CaptureMode::Later,
+                        _ => requests::CaptureMode::Auto,
+                    }),
+                    payment_method: requests::PaymentMethod {
+                        apm: Some(requests::Apm {
+                            provider: Some(requests::ApmProvider::Paypal),
+                        }),
+                        authentication: None,
+                        bank_transfer: None,
+                        card: None,
+                        digital_wallet: None,
+                        encryption: None,
+                        entry_mode: Default::default(),
+                        fingerprint_mode: None,
+                        first_name: None,
+                        id: None,
+                        last_name: None,
+                        name: None,
+                        narrative: None,
+                        storage_mode: None,
+                    },
+                    notifications: Some(requests::Notifications {
+                        return_url: item.router_return_url.clone(),
+                        challenge_return_url: None,
+                        decoupled_challenge_return_url: None,
+                        status_url: None,
+                        three_ds_method_return_url: None,
+                    }),
+                    authorization_mode: None,
+                    cashback_amount: None,
+                    channel: Default::default(),
+                    convenience_amount: None,
+                    currency_conversion: None,
+                    description: None,
+                    device: None,
+                    gratuity_amount: None,
+                    initiator: None,
+                    ip_address: None,
+                    language: None,
+                    lodging: None,
+                    order: None,
+                    payer_reference: None,
+                    site_reference: None,
+                    stored_credential: None,
+                    surcharge_amount: None,
+                    total_capture_count: None,
+                    globalpay_payments_request_type: None,
+                    user_reference: None,
                 }),
-                ..Default::default()
+                api_models::payments::WalletData::GooglePay(_) => Ok(Self {
+                    account_name,
+                    amount: Some(item.request.amount.to_string()),
+                    currency: item.request.currency.to_string(),
+                    reference: item.attempt_id.to_string(),
+                    country: item.get_billing_country()?,
+                    capture_mode: item.request.capture_method.map(|cap_mode| match cap_mode {
+                        enums::CaptureMethod::Manual => requests::CaptureMode::Later,
+                        _ => requests::CaptureMode::Auto,
+                    }),
```

Comment:
add a `From` implementation for this and use it wherever needed, or move it as variable to method start and use it in all other places

Reply:
Done